package.json

```diff
@@ -2,8 +2,12 @@
   "name": "@freecodecamp/github-tools",
   "description": "Tools to help maintain freecodecamp.org's Open Source Codebase on GitHub",
   "private": true,
+  "bin": {
+    "presolver": "probot/index.js"
```

Comment:
I don't think this will work -- I think we need to install the probot on the repository using github apps and maintain it separately.